kf/bpf.go

```diff
@@ -206,6 +211,51 @@ func StopExternalRunningProcess(processName string) error {
 	return nil
 }
 
+// Name returns the program name
+func (b *BPF) Name() string {
+	return b.Program.Name
+}
+
+// Artfact returns the program artifact
+func (b *BPF) Artifact() string {
+	return b.Program.Artifact
+}
+
+// MapName returns the map name for the program
+func (b *BPF) MapName() string {
+	return b.Program.MapName
+}
+
+// MapNamePth returns the map name path
+func (b *BPF) MapNamePth() string {
```

Comment:
Why "Pth" instead of "Path"?

Reply:
We can't have member functions and fields of the same name (go does not allow that). It does feel a little uncanny and we can probably use a better name.